web/web/templates/layouts/base.html

```diff
@@ -154,6 +161,7 @@ <h5>Team members:</h5>
 
   <script src="/static/js/jquery.slim.min.js"></script>
   <script src="/static/js/bootstrap.bundle.min.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
```

Comment:
We have made an effort to serve all scripts we're using from the application, rather than relying on external CDNs. Depending on your preference, I'd be happy to either see this file and the font file it references added to `web/web/static`, or since we only use two icons here, we could just inline two static SVGs for light and dark mode to avoid having to include an entire font.